client/blocks/inline-help/popover.jsx

```diff
@@ -96,6 +107,52 @@ class InlineHelpPopover extends Component {
 		);
 	};
 
+	renderChecklistProgress = () => {
+		const { taskStatuses, siteSuffix, translate } = this.props;
+
+		const totalTasks = tasks.length;
+		const numComplete = reduce(
+			tasks,
+			( count, { id, completed: taskComplete } ) =>
+				taskComplete || get( taskStatuses, [ id, 'completed' ] ) ? count + 1 : count,
+			0
+		);
+
+		if ( ! totalTasks ) {
+			return false;
+		}
+
+		if ( numComplete === totalTasks ) {
+			// @TODO show something else?
+			return false;
+		}
+		const checklistLink = '/checklist' + siteSuffix;
+
+		return (
+			<div className="inline-help__checklist">
+				<Button
+					onClick={ this.checklistClicked }
+					href={ checklistLink }
+					className="inline-help__checklist-button"
+					borderless
+				>
+					<span className="inline-help__checklist-label">
+						<Gridicon icon="list-checkmark" className="inline-help__gridicon-list-checkmark" />
+						{ translate( 'Continue Site Setup' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **20** times:
`translate( 'Continue your site setup:' )` *ES Score: 14*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).